bip-musig2.mediawiki

```diff
@@ -514,6 +515,10 @@ The reference implementation is for demonstration purposes only and not to be us
 === Modifications to Nonce Generation ===
 
 Implementers must avoid modifying the ''NonceGen'' algorithm without being fully aware of the implications.
+
+One potential pitfall emerges when ''NonceGen'' is altered such that ''k<sub>1</sub>'' and ''k<sub>2</sub>'' are drawn uniformly at random instead of setting them to the output of a ''hash'' function.
```

Comment:
```suggestion
One potential pitfall emerges when ''NonceGen'' is altered such that ''k<sub>1</sub>'' and ''k<sub>2</sub>'' are drawn uniformly at random instead of setting them to the output of a ''hash'' function that includes ''pk'' as input.
```